status/statusPage.html

```diff
@@ -0,0 +1,699 @@
+<!--- Programmed by Mira Welner at UC Davis 2019 in Professor Jason Lowe-Power's laboratory based on design by Ayaz Akram -->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+    <meta charset="UTF-8">
+    <title>Gem5Art</title>
```

Comment:
All "Gem5" should be "gem5". It's like iPhone, the first letter should always be lower case.